packages/headless/src/api/knowledge/tests/stream-answer-api-state-mock.ts

```diff
@@ -31,6 +31,9 @@ export const streamAnswerAPIStateMock: StateNeededByAnswerAPI = {
       answerConfigurationId: '',
     },
   },
+  insightConfiguration: {
+    insightId: 'allo',
+  },
```

Comment:
nit pick but do you mind adding a better mock value than "allo" ? Someting like "my-insight-id".